tests/playwright/package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "devDependencies": {
+    "@playwright/test": "^1.21.0"
+  },
+  "name": "adyen_salesforce_e2e_tests",
+  "version": "1.0.0",
+  "description": "e2e tests for Adyen SalesForce Cartridges",
+  "main": "index.js",
+  "dependencies": {
```

Comment:
should these be devDependencies?

Reply:
We only install playwright as dev dependency, the rest is added by playwright during installation. I don't have a strong opinion on this.